guides/doc-Provisioning_Guide/topics/Resources.adoc

```diff
@@ -198,6 +198,54 @@ To view the kickstart tree enter the following command:
 # hammer medium list --organization "_your_organization_"
 ----
 
+[[Configuring_Provisioning_Resources-Creating_Installation_Media]]
+=== Using Installation Media
+
+Installation media are sources of packages that {ProjectServer} uses to install a base operating system on a machine from external repository. Installation media must be in the format of an  +operating system installation tree, and must be accessible to the machine hosting the installer through an HTTP URL. You can view installation media by navigating to *Hosts* > *Installation +Media* menu.
+
+{Project} comes with a number of installation media pre-defined to official Linux distribution mirrors. The *Path* should be modified to point to the closest mirror or local copy in order to achieve better download performance when installing multiple hosts. Some installation media can be only used with specific versions, for example *CentOS mirror (7.x)* must be used for CentOS 7 or older and *CentOS mirror (8.x)* must be used for CentOS 8 or higher.
```

Comment:
```suggestion
By default, {Project} includes installation media for some official Linux distributions. Note that some of those installation media are targeted for a specific version of an operating system. For example *CentOS mirror (7.x)* must be used for CentOS 7 or later, and *CentOS mirror (8.x)* must be used for CentOS 8 or later.

If you want to improve download performance when using installation media to install operating systems on multiple host, you must modify the installation medium's *Path* to point to the closest mirror or a local copy. 
```
A suggestion to add structure and improve style.

Reply:
"For example *CentOS mirror (7.x)* must be used for CentOS 7 or later" should be "For example *CentOS mirror (7.x)* must be used for CentOS 7 or earlier". I am gonna fix that once I merge your change.